lib/mayaUsd/python/CMakeLists.txt

```diff
@@ -86,6 +91,10 @@ mayaUsd_compile_config(${PYTHON_TARGET_NAME})
 target_link_libraries(${PYTHON_TARGET_NAME}
     PRIVATE
         ${PROJECT_NAME}
+    PRIVATE
+        Qt5::Core
+        Qt5::Gui
+        Qt5::Widgets
```

Comment:
Sorry I don't like this. We should not make this lib link with Qt. The only lib in our repo that should link with Qt is the mayaUsdUI lib.